presto-hive/src/main/java/com/facebook/presto/hive/HivePartitionManager.java

```diff
@@ -282,6 +277,24 @@ else if (type instanceof TinyintType
                 .orElseThrow(() -> new TableNotFoundException(tableName));
     }
 
+    public static HivePartition parsePartition(
+            SchemaTableName tableName,
+            String partitionId,
+            List<HiveColumnHandle> partitionColumns,
+            List<Type> partitionColumnTypes,
+            DateTimeZone timeZone)
+    {
+        List<String> keys = extractPartitionKeyValues(partitionId);
```

Comment:
s/keys/values - `extractPartitionKeyValues` name is confusing; it suggests that the return value is a key-value maps of partition column names and values; in fact, the return value is a list of values of partition columns